tests/plugins/rpc_command.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+"""
+This plugin is used to test the `rpc_command` hook.
+"""
+import json
+from lightning import Plugin
+
+plugin = Plugin(dynamic=False)
+
+
+@plugin.hook("rpc_command")
+def on_rpc_command(plugin, rpc_command, **kwargs):
+    if rpc_command["method"] == "invoice":
+        # Replace part of this command
+        rpc_command["params"] = json.loads(rpc_command["params"])
```

Comment:
See comment on previous comment regarding serializing the `params` as a string instead of a raw JSON object.